tetris.sh

```diff
@@ -17,13 +17,73 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
 
+# show usage when given -h argument
+usage() {
+    print_languages() {
+        if [ ! -d "lang/" ]; then                                  # check if lang/ exist
+            echo "language(s) does not loaded, please check lang/ directory.";
+        else
+            if [ -z "$(ls lang/ 2> /dev/null)" ]; then             # check if lang/ empty
+                echo "language files not found, please insert those."
+            fi
+        fi        
+        ls -I README.md -1t lang/ 2> /dev/null | cut -d'.' -f1;     # print languages
+    }
+
+    printf "%s\n" \                                                 # prin main usage
+        "Usage: $0 [-h] [-l language]" \
+        "supported languages:" \
+        "english (it is defualt, option not required)" \
+        "$(print_languages)"
+    exit 0;
+}
+
+# BEGIN OF LANGUAGE LINES
+# if you want see or add other languages, visit lang/ dir and read lang/README.md
+# default localized values english:
+local_LinesComplated="Lines completed: ";
```

Comment:
typo: Complated -> Completed.

Please also note my earlier comment on the consistent code style.